s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -12,12 +25,15 @@ import (
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/opensds/multi-cloud/api/pkg/common"
 	. "github.com/opensds/multi-cloud/s3/error"
-	helper "github.com/opensds/multi-cloud/s3/pkg/helper"
+	"github.com/opensds/multi-cloud/s3/pkg/helper"
 	. "github.com/opensds/multi-cloud/s3/pkg/meta/types"
+	"github.com/opensds/multi-cloud/s3/pkg/utils"
 	pb "github.com/opensds/multi-cloud/s3/proto"
 	log "github.com/sirupsen/logrus"
 )
 
+const TimeDur = 5000 // milisecond
```

Comment:
what is this for ?

Reply:
We want to print a log if one select sql command runs more than 5000 miliseconds.